code/modules/vehicles/cm_armored.dm

```diff
@@ -408,6 +417,10 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 		var/obj/structure/table/T = A
 		T.visible_message("<span class='danger'>[root] crushes [T]!</span>")
 		T.destroy_structure(TRUE)
+	else if(istype(A, /obj/structure/closet))
```

Comment:
If you are willing, you could also do a generic behavior for structures along the lines of
`if(breakable) then C.destroy_structure(TRUE)`

Else this should be ready to merge.